x-pack/plugins/secops/public/components/timeline/data_providers/data_providers.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import { DataProvider } from './data_provider';
+import { DataProviders } from './data_providers';
+import {
+  getEventCount,
+  mockDataProviderNames,
+  mockDataProviders,
+} from './mock/mock_data_providers';
+
+describe('DataProviders', () => {
+  describe('rendering', () => {
+    test('it should render a placeholder when there are zero data providers', () => {
+      const dataProviders: DataProvider[] = [];
+
+      const wrapper = mount(
+        <DataProviders dataProviders={dataProviders} onDataProviderRemoved={fp.noop} />
+      );
+
+      expect(wrapper.text()).toContain('Drop anything with a Facet count here');
```

Comment:
Seems like your style you would have made this text label a const to not repeat your text copy-paste in two places here.

```ts
const DROP_TEXT = 'Drop anything with a Facet count here'
```

No changes requested, just an observation